CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
+## [v9.13.0](https://github.com/auth0/auth0.js/tree/v9.13.0) (2020-03-27)
```

Comment:
There are at least 2 more PRs (Security, docs, tests) tagged with CH labels but not put into this milestone. Did you leave them out of the changelog on purpose?

Reply:
No these should be in - I've amended the changelog. Thanks!